api/apps/brand_partners/model.coffee

```diff
@@ -0,0 +1,83 @@
+#
+# Library of retrieval, persistance, validation, json view, and domain logic
+# for the "brand-partners" resource.
+#
+
+_ = require 'underscore'
+db = require '../../lib/db'
+async = require 'async'
+Joi = require 'joi'
+Joi.objectId = require 'joi-objectid'
+{ ObjectId } = require 'mongojs'
+{ ARTSY_URL, API_MAX, API_PAGE_SIZE } = process.env
+
+#
+# Schemas
+#
+schema = (->
+  id: @objectId()
+  partner_id: @objectId()
+  slug: @string().allow('', null)
```

Comment:
I don't think this will be necessary b/c the slug is determined by the partner model.